test/core/Gatekeeping.t.sol

```diff
@@ -309,6 +307,25 @@ contract GatekeepingTest is MangroveTest {
     tkr.marketOrder(0, 1 << 160);
   }
 
+  /*
+  // FIXME reactivate this test
```

Comment:
There are tests of `clean` parameter validation tests in `TakerOperations.t.sol` , eg `test_gives_volume_above_96bits_fails_clean`.

So not sure these tests should be reactivated?

If they should, they should be placed together with those other clean parameter validation tests.

Reply:
removed!